[2024-07-17T12:54:45.472+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:54:45.472+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:54:45.473+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:54:45.473+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:54:45.668+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:54:45.665+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:54:45.669+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:54:45.689+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.220 seconds
[2024-07-17T12:54:50.287+0000] {processor.py:161} INFO - Started process (PID=109) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:54:50.287+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:54:50.288+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:54:50.287+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:54:50.510+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:54:50.506+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:54:50.511+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:54:50.533+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.249 seconds
[2024-07-17T12:55:20.760+0000] {processor.py:161} INFO - Started process (PID=112) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:55:20.761+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:55:20.762+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:55:20.761+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:55:21.045+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:55:21.041+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:55:21.047+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:55:21.069+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.311 seconds
[2024-07-17T12:55:51.232+0000] {processor.py:161} INFO - Started process (PID=115) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:55:51.232+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:55:51.233+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:55:51.233+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:55:51.428+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:55:51.424+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:55:51.429+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:55:51.449+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.220 seconds
[2024-07-17T12:56:21.696+0000] {processor.py:161} INFO - Started process (PID=118) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:56:21.696+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:56:21.697+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:56:21.697+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:56:21.878+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:56:21.875+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:56:21.879+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:56:21.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.206 seconds
[2024-07-17T12:56:52.147+0000] {processor.py:161} INFO - Started process (PID=121) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:56:52.148+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:56:52.148+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:56:52.148+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:56:52.340+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:56:52.337+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:56:52.341+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:56:52.361+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.217 seconds
[2024-07-17T12:57:22.602+0000] {processor.py:161} INFO - Started process (PID=124) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:57:22.602+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:57:22.603+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:57:22.603+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:57:22.794+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:57:22.790+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:57:22.795+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:57:22.825+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.226 seconds
[2024-07-17T12:57:53.005+0000] {processor.py:161} INFO - Started process (PID=127) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:57:53.006+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:57:53.007+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:57:53.006+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:57:53.193+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:57:53.190+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:57:53.195+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:57:53.214+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.212 seconds
[2024-07-17T12:58:23.470+0000] {processor.py:161} INFO - Started process (PID=130) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:58:23.471+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:58:23.471+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:58:23.471+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:58:23.650+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:58:23.647+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:58:23.651+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:58:23.679+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.212 seconds
[2024-07-17T12:58:53.947+0000] {processor.py:161} INFO - Started process (PID=133) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:58:53.947+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:58:53.948+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:58:53.948+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:58:54.147+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:58:54.143+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:58:54.148+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:58:54.177+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.233 seconds
[2024-07-17T12:59:24.436+0000] {processor.py:161} INFO - Started process (PID=136) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:59:24.436+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:59:24.437+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:59:24.437+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:59:24.623+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:59:24.620+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:59:24.624+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:59:24.653+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.220 seconds
[2024-07-17T12:59:54.891+0000] {processor.py:161} INFO - Started process (PID=139) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:59:54.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T12:59:54.892+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:59:54.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:59:55.085+0000] {logging_mixin.py:188} INFO - [2024-07-17T12:59:55.081+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T12:59:55.086+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T12:59:55.113+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.225 seconds
[2024-07-17T13:00:25.366+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:00:25.366+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:00:25.367+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:00:25.367+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:00:25.577+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:00:25.573+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:00:25.579+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:00:25.610+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.248 seconds
[2024-07-17T13:00:55.833+0000] {processor.py:161} INFO - Started process (PID=145) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:00:55.833+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:00:55.834+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:00:55.834+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:00:56.008+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:00:56.004+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:00:56.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:00:56.028+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.198 seconds
[2024-07-17T13:01:26.304+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:01:26.305+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:01:26.305+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:01:26.305+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:01:26.495+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:01:26.491+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:01:26.496+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:01:26.517+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.216 seconds
[2024-07-17T13:01:56.761+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:01:56.762+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:01:56.762+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:01:56.762+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:01:56.959+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:01:56.956+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:01:56.960+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:01:56.988+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.231 seconds
[2024-07-17T13:02:27.204+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:02:27.204+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:02:27.205+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:02:27.205+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:02:27.405+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:02:27.402+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:02:27.406+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:02:27.425+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.224 seconds
[2024-07-17T13:02:57.677+0000] {processor.py:161} INFO - Started process (PID=157) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:02:57.677+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:02:57.678+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:02:57.677+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:02:57.865+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:02:57.862+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:02:57.866+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:02:57.885+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.212 seconds
[2024-07-17T13:03:28.126+0000] {processor.py:161} INFO - Started process (PID=160) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:03:28.127+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:03:28.127+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:03:28.127+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:03:28.321+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:03:28.318+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:03:28.322+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:03:28.341+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.219 seconds
[2024-07-17T13:03:58.585+0000] {processor.py:161} INFO - Started process (PID=163) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:03:58.585+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:03:58.586+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:03:58.585+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:03:58.776+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:03:58.772+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:03:58.777+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:03:58.807+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.226 seconds
[2024-07-17T13:04:29.063+0000] {processor.py:161} INFO - Started process (PID=166) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:04:29.064+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:04:29.064+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:04:29.064+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:04:29.256+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:04:29.252+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:04:29.257+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:04:29.277+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.217 seconds
[2024-07-17T13:04:59.532+0000] {processor.py:161} INFO - Started process (PID=169) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:04:59.532+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:04:59.533+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:04:59.533+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:04:59.726+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:04:59.722+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:04:59.727+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:04:59.756+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.228 seconds
[2024-07-17T13:05:29.981+0000] {processor.py:161} INFO - Started process (PID=172) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:05:29.981+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:05:29.982+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:05:29.982+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:05:30.168+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:05:30.164+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:05:30.169+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:05:30.196+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.218 seconds
[2024-07-17T13:06:00.436+0000] {processor.py:161} INFO - Started process (PID=175) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:06:00.437+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:06:00.438+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:06:00.437+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:06:00.619+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:06:00.615+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:06:00.620+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:06:00.638+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.204 seconds
[2024-07-17T13:06:30.917+0000] {processor.py:161} INFO - Started process (PID=178) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:06:30.918+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:06:30.918+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:06:30.918+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:06:31.116+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:06:31.113+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:06:31.117+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:06:31.146+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.232 seconds
[2024-07-17T13:07:01.376+0000] {processor.py:161} INFO - Started process (PID=181) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:07:01.377+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:07:01.377+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:07:01.377+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:07:01.567+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:07:01.564+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:07:01.568+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:07:01.591+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.218 seconds
[2024-07-17T13:07:31.853+0000] {processor.py:161} INFO - Started process (PID=184) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:07:31.854+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:07:31.854+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:07:31.854+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:07:32.046+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:07:32.043+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:07:32.047+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:07:32.067+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.217 seconds
[2024-07-17T13:08:02.321+0000] {processor.py:161} INFO - Started process (PID=187) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:08:02.322+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:08:02.322+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:08:02.322+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:08:02.517+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:08:02.514+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:08:02.518+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:08:02.537+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.219 seconds
[2024-07-17T13:08:32.773+0000] {processor.py:161} INFO - Started process (PID=190) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:08:32.773+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:08:32.774+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:08:32.774+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:08:32.955+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:08:32.951+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:08:32.956+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:08:32.984+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.215 seconds
[2024-07-17T13:09:03.212+0000] {processor.py:161} INFO - Started process (PID=193) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:09:03.212+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:09:03.213+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:09:03.213+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:09:03.398+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:09:03.395+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:09:03.399+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:09:03.417+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.208 seconds
[2024-07-17T13:09:33.625+0000] {processor.py:161} INFO - Started process (PID=196) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:09:33.625+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:09:33.626+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:09:33.625+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:09:33.814+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:09:33.810+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:09:33.815+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:09:33.836+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.214 seconds
[2024-07-17T13:10:04.084+0000] {processor.py:161} INFO - Started process (PID=199) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:10:04.085+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:10:04.085+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:10:04.085+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:10:04.277+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:10:04.274+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:10:04.278+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:10:04.297+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.216 seconds
[2024-07-17T13:10:34.538+0000] {processor.py:161} INFO - Started process (PID=202) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:10:34.539+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:10:34.540+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:10:34.540+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:10:34.742+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:10:34.739+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:10:34.744+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:10:34.765+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.230 seconds
[2024-07-17T13:11:05.031+0000] {processor.py:161} INFO - Started process (PID=205) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:11:05.032+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:11:05.033+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:11:05.032+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:11:05.297+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:11:05.293+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:11:05.298+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:11:05.318+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.296 seconds
[2024-07-17T13:11:35.505+0000] {processor.py:161} INFO - Started process (PID=208) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:11:35.506+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:11:35.506+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:11:35.506+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:11:35.767+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:11:35.763+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:11:35.768+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:11:35.790+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.289 seconds
[2024-07-17T13:12:05.953+0000] {processor.py:161} INFO - Started process (PID=211) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:12:05.954+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:12:05.955+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:12:05.954+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:12:06.156+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:12:06.153+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:12:06.157+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:12:06.180+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.230 seconds
[2024-07-17T13:12:36.440+0000] {processor.py:161} INFO - Started process (PID=214) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:12:36.440+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:12:36.441+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:12:36.441+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:12:36.629+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:12:36.626+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:12:36.631+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:12:36.651+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.214 seconds
[2024-07-17T13:13:06.915+0000] {processor.py:161} INFO - Started process (PID=217) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:13:06.916+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:13:06.916+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:13:06.916+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:13:07.110+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:13:07.106+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:13:07.111+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:13:07.131+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.220 seconds
[2024-07-17T13:13:37.404+0000] {processor.py:161} INFO - Started process (PID=220) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:13:37.405+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:13:37.406+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:13:37.405+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:13:37.623+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:13:37.619+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:13:37.624+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:13:37.644+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.243 seconds
[2024-07-17T13:14:07.891+0000] {processor.py:161} INFO - Started process (PID=223) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:14:07.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:14:07.892+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:14:07.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:14:08.085+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:14:08.081+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:14:08.086+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:14:08.106+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.219 seconds
[2024-07-17T13:14:38.382+0000] {processor.py:161} INFO - Started process (PID=226) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:14:38.383+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:14:38.383+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:14:38.383+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:14:38.578+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:14:38.574+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:14:38.579+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:14:38.599+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.220 seconds
[2024-07-17T13:15:08.811+0000] {processor.py:161} INFO - Started process (PID=229) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:15:08.812+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:15:08.813+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:15:08.812+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:15:09.008+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:15:09.005+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:15:09.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:15:09.037+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.229 seconds
[2024-07-17T13:15:39.240+0000] {processor.py:161} INFO - Started process (PID=232) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:15:39.240+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:15:39.241+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:15:39.241+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:15:39.442+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:15:39.438+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:15:39.443+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:15:39.473+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.237 seconds
[2024-07-17T13:16:09.665+0000] {processor.py:161} INFO - Started process (PID=235) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:16:09.665+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:16:09.666+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:16:09.666+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:16:09.862+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:16:09.858+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:16:09.863+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:16:09.884+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.223 seconds
[2024-07-17T13:16:40.086+0000] {processor.py:161} INFO - Started process (PID=238) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:16:40.087+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:16:40.087+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:16:40.087+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:16:40.302+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:16:40.298+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:16:40.303+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:16:40.323+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.241 seconds
[2024-07-17T13:17:10.499+0000] {processor.py:161} INFO - Started process (PID=241) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:17:10.500+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:17:10.501+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:17:10.500+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:17:10.725+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:17:10.721+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:17:10.726+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:17:10.748+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.252 seconds
[2024-07-17T13:17:40.929+0000] {processor.py:161} INFO - Started process (PID=244) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:17:40.930+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:17:40.930+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:17:40.930+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:17:41.132+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:17:41.128+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:17:41.133+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:17:41.163+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.237 seconds
[2024-07-17T13:18:11.369+0000] {processor.py:161} INFO - Started process (PID=247) to work on /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:18:11.370+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_for_kurs.py for tasks to queue
[2024-07-17T13:18:11.370+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:18:11.370+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:18:11.565+0000] {logging_mixin.py:188} INFO - [2024-07-17T13:18:11.561+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_for_kurs.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_for_kurs.py", line 5, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-07-17T13:18:11.566+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_for_kurs.py
[2024-07-17T13:18:11.586+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_for_kurs.py took 0.220 seconds
